tomography_preprocessing/tilt_series_alignment/imod/alignment.py

```diff
@@ -68,5 +68,3 @@ def align_single_tilt_series(
         imod_directory=alignment_dir,
         output_star_file=image_dir / tilt_image_metadata_filename,
     )
-    console.save_text(alignment_dir / 'log.txt', clear=False)
```

Comment:
Removed this in the alignment functions as it was overwriting the logs from the batch function.